content.md

```diff
@@ -705,8 +705,12 @@ cards. When this template is used, only one Y can be done per X with
 this card. For instance, you may move blood from your vampire to your
 pool with Vessel only once per unlock phase.
 
-**"Lock X to do Y":** These effects require an unlocked minion to be
-used, they cannot be used by locked minions.
+**"Cost X to do Y":** The "cost X to do Y" template is used on several
+cards. A minion or Methuselah playing a card or using an effect with
+this template will have to pay cost X in order to play the card or use
+the effect Y. For example, an effect with "lock a location to lock
+a minion" require the Methuselah to lock an unlocked location they
+control to lock a minion.
```

Comment:
I think a change is required, but I have minor issues with this proposal:

1. The "lock X to" is important and has to stay clear and upfront because it is easy to argue you can lock a locked thing and it just stays locked. 
2. It should be kept short: the first sentence can be removed and the second, as well as the first half of the second.
3. I don't think the provided example exists as actual card text, we should shy away from using imaginary examples in the rulebook in my opinion.

Still convoluted but maybe:
```suggestion
**"Do X to do Y":** These effects require X to be played. For example "Lock this vampire to do Y" requires to lock the unlocked vampire: it cannot be used by a locked vampire. "Burn the Edge to" requires the Edge, "burn a blood to" requires a blood, and so on.
```

Alternatively:
```suggestion
**"Lock X to do Y":** These effects require an unlocked minion to be used, they cannot be used by locked minions. Similarily, **"Do X to do Y"** effects require X and cannot be used without X. "Burn the Edge to" requires the Edge, "Burn a blood to" requires a blood, and so on.
```